main/boost/APKBUILD

```diff
@@ -15,7 +15,7 @@ source="https://dl.bintray.com/boostorg/release/$pkgver/source/${pkgname}_$_pkgv
 	boost-1.57.0-python-abi_letters.patch
 	boost-1.57.0-python-libpython_dep.patch
 	revert-wifsignaled-static-assert.patch
-	disable-debugger-test.patch
```

Comment:
you need to add removal of the file to commit

Reply:
Thanks...I have removed it